src/components/structure/interacting/my-cranach/my-cranach.tsx

```diff
@@ -1,37 +1,25 @@
 import React, { useContext } from 'react';
 import { observer } from 'mobx-react-lite';
-
 import Size from '../../../base/visualizing/size';
-
 import translations from './translations.json';
 import './my-cranach.scss';
 
 import StoreContext from '../../../../store/StoreContext';
 
-
 const MyCranach = () => {
   const { collection, ui } = useContext(StoreContext);
   const { t } = ui.useTranslation('SearchSidebar', translations);
-  const showCollection = () => collection?.showCollection();
-  const triggerComparism = () =>  collection?.startComparism();
+  const triggerComparism = () => collection?.startComparism();
```

Comment:
Das `?` ist mit der Anpassung des StoreContext neulich nicht mehr nötig. In Zeile 20 ebenfalls nicht mehr.